examples/lakers-no_std/src/main.rs

```diff
@@ -6,15 +6,16 @@ use cortex_m_rt::entry;
 use cortex_m_semihosting::debug::{self, EXIT_SUCCESS};
 
 #[cfg(not(feature = "rtt"))]
```

Comment:
I think we should select based on the target, like this: 

```rust
#[cfg(target = "thumbv7m-none-eabi")]
```

Reply:
Looks like target doesn't work. [We don't have this option](https://doc.rust-lang.org/reference/conditional-compilation.html#set-configuration-options)

I used `#[cfg(target_abi = "eabihf")]` and `#[cfg(not(target_abi = "eabihf"))]` if this is fine

Reply:
Oh it's true. Looks good then!